docs/constraints.md

```diff
@@ -32,13 +32,13 @@ The following definitions capture the notion of triple-term-subject well-formedn
 
 **Definition:** A triple term (*s*, *p*, *o*) is **triple-term-subject well-formed** if it has both of the following two properties:
 
-1. *s* is a not a triple term, and
-2. if *o* is a triple term, then *o* is triple-term-subject well-formed.
+* *s* is a not a triple term
+* *o* is a triple term
 
 **Definition:** An RDF triple (*s*, *p*, *o*) is **triple-term-subject well-formed** if it has both of the following two properties:
 
-1. *s* is a not a triple term, and
-2. if *o* is a triple term, then *o* is triple-term-subject well-formed.
+* *s* is a not a triple term
+* *o* is a triple term
```

Comment:
This is the only change that I didn't replicate in PR #123 because it changes the meaning of the definition. That is, saying only that _o_ needs to be a triple term has the following two issues:
* It rules out the option that _o_ may be something other than a triple term (which, in fact, would mean that there can be no triples that do not contain a triple term).
* For the case that _o_ is indeed a triple term, the definition needs to be recursive; that is, this triple term must itself be triple-term-subject well-formed.

Reply:
I've now re-read this several times, and *think* I understand what you're getting at. See if this addresses your point --
```suggestion
* *o* either is not a triple term, or is a triple term which is itself triple-term-subject well-formed
```

Reply:
That could be modified to the following --
```suggestion
* either 
  * *o* is not a triple term
  * *o* is a triple term which is itself triple-term-subject well-formed
```

Reply:
You understood correctly. I implemented this edit suggestion in PR #123 (see, in particular, commit https://github.com/w3c/rdf-star-wg/pull/123/commits/6a6d73d200e6589fe86a1da3f97d66d1cb579a8e). If you approve that other PR, I will merge it.